connector/connect/server/src/test/scala/org/apache/spark/sql/connect/execution/ReattachableExecuteSuite.scala

```diff
@@ -40,8 +40,7 @@ class ReattachableExecuteSuite extends SparkConnectServerTest {
       val reattachableIter = getReattachableIterator(iter)
       val initialInnerIter = reattachableIter.innerIterator
 
-      // open the iterator
-      iter.next()
+      iter.next() // open iterator, guarantees that the RPC reached the server
```

Comment:
The bunch of opening the iterator is still needed in these tests. It's an easy way to assert that the query has actually reached the server already, without having to resort to an Eventually check of server state.